### PR TITLE
Feature/gui highlight changed register

### LINKED ISSUE #182: Highlight last changed values

### PR DESCRIPTION
**Pull request by riedersa**
Closes #182

### CODE REVIEW

File: source/ui/Components/Register/DefaultRegister.qml [hunk 1]
```diff
@@ -64,6 +64,7 @@ TextField {
             // Check if the current item's index is affected by the data change.
             if (topLeft <= styleData.index && styleData.index <= bottomRight) {
                 text = Qt.binding(registerContent);
+                style = styleChanged;
```
Comment (FelixOpolka) [inline source/ui/Components/Register/DefaultRegister.qml:68]: Maybe add a comment here to explain what this does.

File: source/ui/Components/Register/FlagRegister.qml [hunk 2]
```diff
@@ -37,6 +38,7 @@ CheckBox {
             // Check if the current item's index is affected by the data change.
             if (topLeft <= styleData.index && styleData.index <= bottomRight) {
                 checked = Qt.binding(function() {return model.FlagData});
+                registerCheckBox.style = styleChanged;
```
Comment (FelixOpolka) [inline source/ui/Components/Register/FlagRegister.qml:42]: Maybe add a comment here to explain what this does.

File: source/ui/Components/Register/DefaultRegister.qml [hunk 3]
```diff
@@ -95,4 +96,63 @@ TextField {
...
+                focus: false
+                sourceComponent: glowEffect
+            }
+
+            Component {
+                id: glowEffect
+                Item{
+                    Rectangle{
+                        id: rect
+                        x: registerTextField.x; y: registerTextField.y
```
Comment (FelixOpolka) [inline source/ui/Components/Register/DefaultRegister.qml]: I'd suggest to put these in two separate lines to have a uniform style among our files; but that's just a really minor thing.

**Review by FelixOpolka — APPROVED** [review]

**Comment by riedersa** [comment]
As told in last Meeting we Need an other implementation

**Pull request closed without merging**